src/pages/fr/install/auto.mdx

```diff
@@ -1,8 +1,6 @@
 ---
 title: Installation d'Astro avec l'ILC automatique
-description: >-
-  Comment installer Astro avec NPM, PNPM, ou Yarn via l'outil de création
-  create-astro inclus dans l'ILC.
+description: "Comment installer Astro avec NPM, PNPM, ou Yarn via l'outil de création create-astro inclus dans l'ILC."
 layout: ~/layouts/MainLayout.astro
 ---
 import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
```

Comment:
Thanks for this @MoustaphaDev!

Heads up that we just made a couple of small changes to this file in https://github.com/withastro/docs/commit/35291728a07e26a0beb6ffcfa609ad83873217a5#diff-8e1312aaccefef4740d3ffd0315425a4f9f65c314f7821a64754f90036b05388 

Sorry — sometimes Astro just moves to fast 😅